client/public/index.html

```diff
@@ -11,31 +11,28 @@
 
     <link rel="icon" href="%PUBLIC_URL%/favicon-48x48.png" />
 
-    <!-- third-generation iPad with high-resolution Retina display: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      sizes="144x144"
-      href="%PUBLIC_URL%/favicon144.png"
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="%PUBLIC_URL%/apple-touch-icon.png"
     />
-
-    <!-- iPhone with high-resolution Retina display: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      sizes="114x114"
-      href="%PUBLIC_URL%/favicon114.png"
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="%PUBLIC_URL%/favicon-32x32.png"
     />
-
-    <!-- first- and second-generation iPad: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      sizes="72x72"
-      href="%PUBLIC_URL%/favicon72.png"
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="%PUBLIC_URL%/favicon-16x16.png"
     />
-
-    <!-- non-Retina iPhone, iPod Touch, and Android 2.1+ devices: -->
     <link
-      rel="apple-touch-icon-precomposed"
-      href="%PUBLIC_URL%/favicon57.png"
+      rel="icon"
+      type="image/png"
+      sizes="192x192"
+      href="%PUBLIC_URL%/favicon-192x192.png"
     />
```

Comment:
Is it meant for Android devices? If so, this is unnecessary. Both Chrome and Firefox use apple-touch-icon or pick ones defined in manifest.json.

Reply:
Technically yes but, https://realfavicongenerator.net/blog/android-chrome-and-its-favicon/ - Perhaps this is no longer an issue though? Would be good to know.

Reply:
It looks like HTML5 boilerplate has literally scaled down to just two lines:
https://github.com/h5bp/html5-boilerplate/blob/master/src/index.html#L15

And then the 192x192 is specified in the webmanifest, with the `.ico` at the root so, we can follow suit?

Reply:
> Technically yes but, https://realfavicongenerator.net/blog/android-chrome-and-its-favicon/ - Perhaps this is no longer an issue though? Would be good to know.

That post is from 2014 so there should be changes made to browsers... I put the HTML up on https://valley-rainbow-gem.glitch.me/trimmed.html and both Chrome on Android and Fenix utilize the 180 file.

> It looks like HTML5 boilerplate has literally scaled down to just two lines:
> https://github.com/h5bp/html5-boilerplate/blob/master/src/index.html#L15
> 
> And then the 192x192 is specified in the webmanifest, with the `.ico` at the root so, we can follow suit?

so the simplest setup is

* a favicon.ico at root directory
* an apple-touch-icon in the link tag
* (a web app manifest file If you want some PWA experience)

but as per https://github.com/mdn/yari/issues/3611#issuecomment-825847727 it appears you cannot set longer cache TTL to the `/favicon.ico` URL. that's why there's a smaller 48x48 PNG file that has longer cache lifetime here.

Reply:
So @myakura so, what your test indicates is that we can essentially go with:

1. favicon.ico at the root (if we really want, for IE lte10)
2. The `apple-touch-icon` that we can cache with a long TTL (all modern(ish) browsers will pick this one)

And that is pretty much it?

I would also like to, in the very near future, add an SVG for modern browsers with embedded CSS to respond to light and dark modes. But that is separate from this PR.

Reply:
> an apple-touch-icon in the link tag

Both? `<link rel="apple-touch-icon" href="icon.png">` AND `<link rel="icon" href="icon.png">` where `icon.png` is the URL (with a hash in it). 

There's just something "stinky" about `apple-touch-icon` in that it's called "apple". It's a great company with great products but it's a company, not a generic web "standard" thing.

Reply:
>     1. favicon.ico at the root (if we really want, for IE lte10)
> 
>     2. The `apple-touch-icon` that we can cache with a long TTL (all modern(ish) browsers will pick this one)
> 
> 
> And that is pretty much it?

the `apple-touch-icon` is for mobile browsers and with that rel value browsers wont pick it up and show it in the tab. we need `<link rel="icon" href="favicon.png">`.

so we need three files:

1. /favicon.ico at root (an ICO file, for legacy browsers)
2. `<link rel="icon" href="favicon-48.png">` (a PNG file, for modern browsers)
3. `<link rel="apple-touch-icon" href="icon-180.png">` (a PNG file, for mobile browsers)

and two link tags for PNG files.

```html
<link rel="icon" href="favicon-48x48.png">
<link rel="apple-touch-icon" href="icon-180x180.png">
```

> I would also like to, in the very near future, add an SVG for modern browsers with embedded CSS to respond to light and dark modes. But that is separate from this PR.

Cool!